src/Exceptionless.Core/Settings.cs

```diff
@@ -250,6 +244,7 @@ public class Settings {
             settings.AzureStorageQueueConnectionString = configRoot.GetConnectionString("AzureStorageQueue");
             settings.AliyunStorageConnectionString = configRoot.GetConnectionString("AliyunStorage");
             settings.MinioStorageConnectionString = configRoot.GetConnectionString("MinioStorage");
+            settings.MetricsConnectionString = Utility.MetricsConnectionString.Parse(configRoot.GetConnectionString("Metrics"));
```

Comment:
This should just set the connection string and the parsing should happen in the insulation project. The core project shouldn't know anything about statsd or any other implementation.